diff --git a/lib/anime.esm.js b/lib/anime.esm.js
index 7dcc0e50156ee4188e73244ae7fafee7019e6c1d..c7d07715ed3cda72293add0dc081dc31768a01f4 100644
--- a/lib/anime.esm.js
+++ b/lib/anime.esm.js
@@ -762,10 +762,10 @@ const globals = {
 
 const globalVersions = { version: '4.1.4', engine: null };
 
-if (isBrowser) {
-  if (!win.AnimeJS) win.AnimeJS = [];
-  win.AnimeJS.push(globalVersions);
-}
+// if (isBrowser) {
+//   if (!win.AnimeJS) win.AnimeJS = [];
+//   win.AnimeJS.push(globalVersions);
+// }
 
 // Strings
 
@@ -1641,10 +1641,10 @@ const engine = /*#__PURE__*/(() => {
   const engine = new Engine(now());
   if (isBrowser) {
     globalVersions.engine = engine;
-    doc.addEventListener('visibilitychange', () => {
-      if (!engine.pauseOnDocumentHidden) return;
-      doc.hidden ? engine.pause() : engine.resume();
-    });
+    // doc.addEventListener('visibilitychange', () => {
+    //   if (!engine.pauseOnDocumentHidden) return;
+    //   doc.hidden ? engine.pause() : engine.resume();
+    // });
   }
   return engine;
 })();
