diff --git a/lib/anime.esm.js b/lib/anime.esm.js
index f257c7f3a2e96db8d2ad92b0a4e5a8ec9fb4a349..3476f93bfe1c44cd0d0ce09407302bc5a5e2d5d4 100644
--- a/lib/anime.esm.js
+++ b/lib/anime.esm.js
@@ -762,10 +762,10 @@ const globals = {
 
 const globalVersions = { version: '4.1.3', engine: null };
 
-if (isBrowser) {
-  if (!win.AnimeJS) win.AnimeJS = [];
-  win.AnimeJS.push(globalVersions);
-}
+// if (isBrowser) {
+//   if (!win.AnimeJS) win.AnimeJS = [];
+//   win.AnimeJS.push(globalVersions);
+// }
 
 // Strings
 
@@ -1641,10 +1641,10 @@ const engine = /*#__PURE__*/(() => {
   const engine = new Engine(now());
   if (isBrowser) {
     globalVersions.engine = engine;
-    doc.addEventListener('visibilitychange', () => {
-      if (!engine.pauseOnDocumentHidden) return;
-      doc.hidden ? engine.pause() : engine.resume();
-    });
+    // doc.addEventListener('visibilitychange', () => {
+    //   if (!engine.pauseOnDocumentHidden) return;
+    //   doc.hidden ? engine.pause() : engine.resume();
+    // });
   }
   return engine;
 })();
